elastic/kibana
# [Fleet] Fix back button when adding integration from multiple integration package

## Description 

Resolve #116244  

The cancel button when adding a new integration from a package with multiple integration should redirect to the integration (AWS billing for example) and not to the main package (AWS)

## How to tests

1.  Go to http://localhost:5601/app/fleet/integrations/aws-1.3.1/add-integration/billing
1. Click on cancel you should go back to AWS Billing .


Not related
@jen-huang Did we considered saving the integration on the package policy, to allow to edit the integration and not the whole package?
